internal/web/app.go

```diff
@@ -0,0 +1,275 @@
+package web
+
+import (
+	"embed"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/fs"
+	"net/http"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/google/uuid"
```

Comment:
we are already bringing `github.com/gofrs/uuid` so adding a new dep for the same thing seems useless? Cant we reuse the existing uuid lib?

Reply:
good call 💪